### PR TITLE
Creating classes

### PR DESCRIPTION
**Pull request by BregornOriginal**
## In this FEATURE I create the following classes

- Person 
- Student
- Teacher

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  def can_use_services?
+    return true if is_of_age? && @parent_permission
```
Comment (BenMukebo) [inline person.rb]: - [x] Great job so far 👏🏻 Every is fine but According to the project requirements, the public method `can_use_services?` should return true if the person is of age `or` has permission from parents. Kindly let's pay more attention, we should use the "or" sign `||` and not the "and" signed `&&` It should be like this:

```ruby
  def can_use_services? 
    is_of_age?() || @parent_permission
  end
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Chrissiku,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
 - All 3 classes implement and has the right file extension. ✔️ 
 - Rubys' inheritance implemented 👍🏻 
- Gitflow was correctly used.  ✔️   
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (BregornOriginal) [reply person.rb]: thank you!

**Review by smunozmo — APPROVED** [review]
Hi @BregornOriginal, 👋 

## Highlights 👏 

- All Classes are set up correctly, congrats on it! 🎉 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**